webrecorder/web-replay-gen
# Build site with barebones index page

- Set up static site generator with [Eleventy](11ty.dev/)
- Read and display custom configuration from single `wrg-config.json` file

### Manual testing
1. Pull this branch and follow instructions from step 3 onward in [README](https://github.com/webrecorder/web-replay-gen/blob/setup-index-page/README.md#3-install-dependencies).
2. Verify that site is generated as expected in `_site` folder.

During local development of this tool you can also create a separate `wrg-config.local.json` file. To point to this file, create a `.env` file with the following:
```
WRG_CONFIG_NAME=wrg-config.local.json
```
And then rebuild with `npm run build` or run the dev server with `npm run serve`.
